### PR TITLE
PLAT-47 Add unsafe connect

### PR DESCRIPTION
**Pull request by RandoMan70**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,13 +3,13 @@ module github.com/FendAiCorp/cruisecontrol-tunnel
 go 1.16
 
 require (
-	github.com/FendAiCorp/cruisecontrol-api v0.0.6-0.20211221115545-8161f66e5636
-	github.com/FendAiCorp/cruisecontrol-common v0.0.9-0.20211215163512-0cd2ac3c762d
+	github.com/FendAiCorp/cruisecontrol-api v0.0.6-0.20211223163612-0857bbb79899
+	github.com/FendAiCorp/cruisecontrol-common v0.0.9-0.20211223161723-9e597fe9571b
+	github.com/FendAiCorp/fendvpn-common v0.0.1
```
Comment (sshaman1101) [inline go.mod:8]: something wrong here

File: internal/httpapi/clients.go [hunk 2]
```diff
@@ -2,6 +2,11 @@ package httpapi
 
 import (
 	"encoding/json"
+	"fmt"
+	"github.com/FendAiCorp/cruisecontrol-common/common"
+	"github.com/FendAiCorp/cruisecontrol-common/xtime"
+	"github.com/google/uuid"
+	"golang.zx2c4.com/wireguard/wgctrl/wgtypes"
```
Comment (sshaman1101) [inline internal/httpapi/clients.go:9]: It would be nice to have em sorted.

File: internal/httpapi/clients.go [hunk 3]
```diff
@@ -2,6 +2,11 @@ package httpapi
 
 import (
 	"encoding/json"
+	"fmt"
+	"github.com/FendAiCorp/cruisecontrol-common/common"
```
Comment (sshaman1101) [inline internal/httpapi/clients.go:6]: wait, there is the `libCommon` import right below.

File: internal/httpapi/clients.go [hunk 4]
```diff
@@ -94,6 +103,104 @@ func (instance *TunnelAPI) ClientConnect(w http.ResponseWriter, r *http.Request)
...
+		settings := instance.runtime.Settings
+
+		tmpl := `[Interface]
+Address = %s/32
+PrivateKey = %s
+
+[Peer]
+PublicKey = %s
+Endpoint = %s:%d
+AllowedIPs = 10.235.0.0/24
```
Comment (sshaman1101) [inline internal/httpapi/clients.go:183]: shouldn't we use the subnet from a config?

File: cmd/vpn-node/main.go [hunk 5]
```diff
@@ -80,7 +80,7 @@ func initServices(runtime *runtime.TunnelRuntime) error {
 
 	keystore, err := federation_keys.NewFsKeystore(runtime.Settings.ManagementKeystore)
 	if err != nil {
-		return err
+		keystore = nil
```
Comment (sshaman1101) [inline cmd/vpn-node/main.go:83]: what do you think if we return an error here and let me fix that?

File: internal/httpapi/clients.go (hunk 4 above)
Comment (RandoMan70) [reply internal/httpapi/clients.go]: It actually must be "all internet" instead ) Thanks, fixed.

File: cmd/vpn-node/main.go (hunk 5 above)
Comment (RandoMan70) [reply cmd/vpn-node/main.go:83]: Feel free to do whatever is good to manage error here. Federation keys must be optional.

Comment (sshaman1101) [reply cmd/vpn-node/main.go]: Fixed.

**Pull request merged**